pkg/sql/opt/exec/execbuilder/testdata/show_trace

```diff
@@ -167,10 +167,6 @@ $trace_query
 commit sql txn  Del /NamespaceTable/30/1/106/107/"kv2"/4/1
-commit sql txn  DelRange /Table/24/1/1/109
-commit sql txn  DelRange /Table/24/1/2/109
-commit sql txn  DelRange /Table/24/1/3/109
-commit sql txn  DelRange /Table/24/1/5/109
```

Comment:
Do you have a clear understanding as to why this went away?

Reply:
Yes, `9b8eb53881ca41d41335271af68d6b9461211bb0` removes the `RemoveAllTableComments` op which was completely redundant and which generated those `DelRange`s.